.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,82 @@
+name: Nightly checks
+
+on:
+  # workflow_dispatch so that it can be triggered manually if needed
+  workflow_dispatch:
+  schedule:
+    - cron: "34 23 * * *"
+
+jobs:
+  e2e-tests:
```

Comment:
Here is an example of a similar run (slightly changed to run on this PR, but the idea should be the same): https://github.com/janus-idp/operator/actions/runs/7960039726/job/21728192711?pr=204#step:11:1